Common/Helpers/ICloudTableProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers
+{
+    public interface ICloudTableProvider
```

Comment:
if you decide to use dependency injection for these, don't forget to add them to startup :)